lib/config.js

```diff
@@ -24,16 +24,19 @@ var fs = require("fs"),
     isAbsolutePath = require("path-is-absolute"),
     isResolvable = require("is-resolvable"),
     validator = require("./config-validator"),
-    pathIsInside = require("path-is-inside");
+    pathIsInside = require("path-is-inside"),
+    find = require("lodash.find");
```

Comment:
Heh no it is okay. Sorry if I came off the wrong way. Was trying to explain what I was thinking in detail.